README.md

```diff
@@ -9,6 +9,9 @@ Before you begin, you must enable the Enhanced Monitoring option for the RDS ins
 * [Building from source](#building-from-source)
 * [Metrics collected by this integration](#metric-groups-collected-by-this-integration)
 
+#### Note: Upgrading from version 0.1.0 (Python 2.7) to version 0.2.0 (Python 3.x)
+Please note that the new version requires AWS lambda handler to be set to `enhanced_rds.lambda_script.lambda_handler`.
```

Comment:
was this a requirement on the AWS side?

Reply:
A mix - implicit relative imports (import from lambda_script) are not allowed in Python 3. Explicit relative imports (import from .lambda_script) do not work in lambda runtime. This is the only working solution I could find, based on https://gist.github.com/gene1wood/06a64ba80cf3fe886053f0ca6d375bc0 . I am open for other suggestions.